### PR TITLE
feat: Add Binder

### PR DESCRIPTION
**Pull request by yangchoo**
Update dockerfile and add Binder badge.

### CODE REVIEW

**Comment by ramu-phanimukla** [comment]
@yangchoo Is this PR still WIP?

**Comment by yangchoo** [comment]
> @yangchoo Is this PR still WIP?

Yea.. binder is launching fine but the notebooks aren't visible currently and I haven't figured out why yet

**Comment by yangchoo** [comment]
At the moment, the binder link loads fine but the notebooks are not being shown.

Looked deeper into this.. looks like there may be some issues with trying to load Binder on a non-master branch. See https://discourse.jupyter.org/t/improve-documentation-for-new-users-not-working-on-the-master-branch/5509/12

Let's see what happens once the sampledata and notebooks are landed on master.

File: README.rst [hunk 1]
```diff
@@ -134,3 +134,5 @@ Read `Contributing <http://transcriptic.readthedocs.io/en/latest/contributing.ht
    :target: http://transcriptic.readthedocs.io/en/latest/?badge=latest
 .. |Code Coverage| image:: https://codecov.io/gh/strateos/transcriptic/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/strateos/transcriptic
+.. |Binder| image:: https://mybinder.org/badge_logo.svg
+  :target: https://mybinder.org/v2/gh/strateos/transcriptic/master
```
Comment (yangchoo) [inline README.rst:138]: try with 
 https://mybinder.org/v2/gh/strateos/transcriptic/SEA-562_add-binder
to see what will be rendered once merged

Comment (dzleidig) [reply README.rst:138]: Does the uploaded version not have the mocked api connection changes?
image
This is pretty cool though - my first time learning about this. It's mainly for Python and other languages offering a REPL?

Comment (yangchoo) [reply README.rst:138]: Yea.. the base image uses the latest version of txpy but we haven't released the latest changes yet.

Yup! Its pretty common in the scientific community. The current demo notebook isn't the best example.. its commonly used in conjunction with plotting libs and other visualizations

**Review by dzleidig — APPROVED** [review]

**Pull request merged**